### DISCUSSION TITLE
Self-hosted organization runner: OCI mountpoint for cgroup not found

### DISCUSSION THREAD

**Discussion by hissingpanda** [community/community#26480] [Actions]
<p>Hi,</p>
<p>I just setup an organization wide runner on one of our servers and am now receiving the following error on a previously working action that was self-hosted by repository:</p>
<pre><code>/usr/bin/docker: Error response from daemon: OCI runtime create failed: container_linux.go:345: starting container process caused "process_linux.go:281: applying cgroup configuration for process caused \"mountpoint for cgroup not found\"": unknown.
</code></pre>
<p>Here is my YAML, it errors out during the action-rsyncer step.</p>
<pre><code>name: Deploy to Y Drive

on:
  push:
    branches:
      - master

jobs:
  build:
    name: Deployment
    # Self Hosted for VPN
    runs-on: self-hosted
    steps:
    # Checkout Repo
    - name: Checkout
      uses: actions/checkout@v2

    # Deploy to Drive
    - name: Deploy to Y Drive
      uses: Pendect/action-rsyncer@v1.1.0
      env:
        DEPLOY_KEY: ${{secrets.DEPLOY_KEY}}
      with:
        flags: '-rlDzg --delete --checksum'
        options: '--exclude=.env --exclude=*.jpeg --exclude=*.rds --exclude=.Rhistory'
        ssh_options: ''
        src: '/'
        dest: '${{secrets.USER}}@${{secrets.SERVER}}:${{secrets.PATH}}'
</code></pre>
<p>I’m not sure what the issue is as the user and group permissions for the runner are the same as they were before. The runner is for Linux X64 architecture and the server is on Ubuntu 18.04.</p>
<p>Any ideas?</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/hissingpanda">@hissingpanda</a> ,</p>
<p>Whether the organization-level self-hosted runner is installed on the same machine with that of the previous repository-level self-hosted runner? Make sure the self-hosted runner has the same configurations with the previous runner, include the <a href="https://help.github.com/en/actions/hosting-your-own-runners/using-a-proxy-server-with-self-hosted-runners" rel="nofollow noopener"><strong>proxy configurations</strong></a> (if any).</p>
<p>Are there other self-hosted runners existing on the same organization or in the repository where the workflow is running?</p>
<p>I noticed you are only using default label " <strong>self-hosted</strong>" to specify the self-hosted runner.<br>
What you need to pay attention to:<br>
When you have installed multiple self-hosted runners (include repository-level and organization-level), these runners may have different configurations. If you only specify the " <strong>self-hosted</strong>" label on a job in your workflow, randomly one of the available " <strong>Idle</strong>" status runner will be assigned to the job, this assigned self-hosted runner may be not the one which has the configurations needed by the workflow run.</p>
<p>So, I strongly recommend you create some specific <a href="https://help.github.com/en/actions/hosting-your-own-runners/using-labels-with-self-hosted-runners" rel="nofollow noopener"><strong>custom labels</strong></a> for each self-hosted runner, so that you can use these labels to distinguish this runner from other runners. Then when you use self-hosted runner in the workflow, you can specify these labels for the self-hosted runner you want to use. This ensures that you have assigned the right self-hosted runner to the workflow.</p>
<p>For example:</p>
<pre><code>jobs:
  build:
    name: Build
    runs-on: [self-hosted, Linux,ForksForTestt,Ubuntu199.10]
    steps:
    . . .
</code></pre>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/6478d5cb04caa5c71e1de33fbfa2267bbff8a9a7.png" data-download-href="/uploads/short-url/ekODnnG1pPyiSRwbQBGrpf4MXFJ.png?dl=1" title="example.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/46307996/181097955-c31d6468-fa7f-4fc6-9605-6d985834bbda.png" alt="example.PNG" data-base62-sha1="ekODnnG1pPyiSRwbQBGrpf4MXFJ" width="690" height="219" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/6478d5cb04caa5c71e1de33fbfa2267bbff8a9a7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">example.PNG</span><span class="informations">748×238 10.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by hissingpanda** [comment]
<p><span class="mention">@brightranThanks</span> for the prompt response! This is all on the same server yes. I added a label and specified that label within the YAML and this same error is occurring. I don’t think it would be the issue though as I removed the old runner when trying to debug and so this is the only active runner on the server now.</p>
<p>I attempted to re-setup the individual repository action-runner, with proper labeling, and this same error is now occurring. So I think it might be related to something that changed in v2.169.1 as I may have been running an older version before - although it seems the runner updates itself when triggered.</p>

**Comment by hissingpanda** [comment]
<p>Actually it seems this specifically an issue with the docker installation. Not sure what has changed.</p>
